client-ts/Microsoft.AspNetCore.SignalR.Client.TS/Microsoft.AspNetCore.SignalR.Client.TS.csproj

```diff
@@ -13,6 +13,7 @@
 
   <ItemGroup>
     <Inputs Include="*.ts;" />
+    <Inputs Remove="UserAgent.ts" />
```

Comment:
? 
I guess VS did it - revert changes to this file.

Reply:
Yeah, idk what happened here